static/app/views/automations/detail.tsx

```diff
@@ -186,7 +188,11 @@ function AutomationDetailContent({automation}: {automation: Automation}) {
                   />
                   <KeyValueTableRow
                     keyName={t('Created by')}
-                    value={createdByUser?.name || createdByUser?.email || t('Unknown')}
+                    value={
+                      automation.createdBy
+                        ? createdByUser?.name || createdByUser?.email || t('Unknown')
```

Comment:
nit: might be nice to pull this string out as `userDisplayName` or something similar, so we can split this logic out from the view code.